src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -349,6 +349,9 @@
 examples:
   - name: Upload all files that end with .py unless blob exists and has been modified since given date.
     text: az storage blob incremental-copy start --source-container MySourceContainer --source-blob MyBlob --source-account-name MySourceAccount --source-account-key MySourceKey --source-snapshot MySnapshot --destination-container MyDestinationContainer --destination-blob MyDestinationBlob
+  - name: Copies an incremental copy of a blob asynchronously. (autogenerated)
```

Comment:
CLI prefer to use active voice in the description. It will be better to use "Copy" not "Copies".